### PR TITLE
Update success modal for transaction and swap

### PR DESCRIPTION
**Pull request by hsurf22**
### Bug

- [x] Updated CHANGELOG.md
- [x] Add PR label
  - Update success modal for "transaction" and "swap"

screenshot from 2019-02-16 01-41-53

screenshot from 2019-02-16 01-42-26

https://youtu.be/9132xr1UUHQ
https://youtu.be/9132xr1UUHQ
https://youtu.be/9132xr1UUHQ
https://youtu.be/9132xr1UUHQ

### CODE REVIEW

**Comment by hsurf22** [comment]
"Check Status on Etherscan" button needs proper link to etherscan.
Please update this before merge.

**Comment by kvhnuke** [comment]
It is not always `etherscan` we have to change the name to the network service name

File: src/containers/ConfirmationContainer/ConfirmationContainer.vue [hunk 1]
```diff
@@ -156,9 +159,9 @@ export default {
     });
   },
   created() {
-    this.$eventHub.$on('showSuccessModal', (message, linkMessage) => {
+    this.$eventHub.$on('showSuccessModal', (message, linkMessage, linkTo) => {
       if (!message) message = null;
-      this.showSuccessModal(message, linkMessage);
+      this.showSuccessModal(message, linkMessage, linkTo);
```
Comment (gamalielhere) [inline src/containers/ConfirmationContainer/ConfirmationContainer.vue]: Please make sure to do this on all instances of `showSuccessModal`

Comment (hsurf22) [reply src/containers/ConfirmationContainer/ConfirmationContainer.vue]: I shouldn't change linkTo value. I added a new prop "etherscanLink" to pass etherscan URL.

Comment (hsurf22) [reply src/containers/ConfirmationContainer/ConfirmationContainer.vue]: Yeah. I tested on Real network and Ropsten Testnet and they all look good!!!

**Pull request merged**